azure-pipelines.yml

```diff
@@ -128,9 +128,29 @@ stages:
           displayName: Build
           condition: succeeded()
           
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+#- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
   - template: eng\common\templates\post-build\post-build.yml
     parameters:
       # Symbol validation isn't being very reliable lately. This should be enabled back
       # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
       enableSymbolValidation: false
+      # Sourcelink validation isn't passing for Arcade due to some regressions. This should be
+      # enabled back once this issue is resolved: https://github.com/dotnet/arcade/issues/2912
+#      enableSourceLinkValidation: false
+#      publishDependsOn:
+#      - Validate
+#      - ValidateSdk
+#      # This is to enable SDL runs part of Post-Build Validation Stage
+#      SDLValidationParameters:
+#        enable: true
+#        continueOnError: false
+#        params: ' -SourceToolsList @("policheck","credscan")
+#        -TsaInstanceURL $(_TsaInstanceURL)
+#        -TsaProjectName $(_TsaProjectName)
+#        -TsaNotificationEmail $(_TsaNotificationEmail)
+#        -TsaCodebaseAdmin $(_TsaCodebaseAdmin)
+#        -TsaBugAreaPath $(_TsaBugAreaPath)
+#        -TsaIterationPath $(_TsaIterationPath)
+#        -TsaRepositoryName "$(TsaRepositoryName)"
```

Comment:
should this variable just be string "templating"?

Reply:
Sample PR in websdk is using string literals. However, I again followed runtime repo and did as they did: https://github.com/dotnet/runtime/blob/385121f62de417c2a2416add568b34880b4c5ea6/eng/pipelines/official/stages/publish.yml#L51

I don't really know what is the "correct" way of doing this.

Reply:
I trust you did the right thing. I recall I reviewed Vijay's PR & saw they hard coded the repository's name instead.  All good.